batch_requests/views.py

```diff
@@ -77,11 +77,15 @@ def get_response(wsgi_request):
     if hasattr(response, 'render') and callable(response.render):
         response = response.render()
 
+    if hasattr(response, '_headers'):
+        response_headers = response._headers
+    else:
+        response_headers = response.headers
```

Comment:
Let's do an explicit django version check so that it's clear why we're testing multiple places.
```
import django
if django.VERSION > (blah, blah, blah):
    ...
```